components/main-chef-wrapper/cmd/diff.go

```diff
@@ -32,7 +32,7 @@ var (
 	pager     bool
 
 	diffCmd = &cobra.Command{
-		Use:                   "diff [POLICYFILE] [--head | --git GIT_REF | POLICY_GROUP | POLICY_GROUP...POLICY_GROUP]",
+		Use:                   "diff [Policyfile] [--head | --git GIT_REF | POLICY_GROUP | POLICY_GROUP...POLICY_GROUP]",
```

Comment:
I think there's probably some more change to Policyfile/policyfile/POLICYFILE usage.  Perhaps a general guide could be: 

- `Policyfile` - referring to a specific file named Policyfile
- `POLICYFILE` - referring to a user-supplied argument that names a policyfile. If referring to the argument in descriptive text, it should also use POLICYFILE.  This keeps it consistent with other argument placeholders across our commands.  You can see this in this example with GIT_REF, POLICY_GROUP, etc. 
- `policyfile` - referring to the non-specific policyfile as in "a policyfile" or "open the policyfile".  

/cc @tas50 @chef/docs-team

Reply:
Thanks. Updating this.